src/app/components/applications/SSV/MyAccount/components/ReactivateCluster/ReactivateCluster.tsx

```diff
@@ -139,8 +139,13 @@ const ReactivateCluster = () => {
                   <Typography className={classes.SsvPrice}
                               style={{ marginBottom: 0 }}>{formatNumberToUi(totalCost.toFixed(18))} SSV</Typography>
                 </Grid>
-                <PrimaryButton text={'Next'} submitFunction={reactivateCluster}
-                               disable={insufficientBalance || customPeriod <= 0 || isNaN(customPeriod)}/>
+                <Button
+                    withAllowance
+                    text={'Next'}
+                    onClick={reactivateCluster}
+                    disable={insufficientBalance || customPeriod <= 0 || isNaN(customPeriod)}
```

Comment:
Move this complex condition to the same section of component where all the rest of calculations/flags defined and use here only 1 variable

Reply:
done